uwblueprint/community-fridge-kw
# feat: admin view check-in page

## Brief description. What is this change? 

### [Admin View Checkin Page](https://www.notion.so/uwblueprintexecs/Admin-View-Checkin-Page-3198bc8a9fb14ca79e5a654d0af5bbe2)
![80b1894f-0ff3-4948-a469-147f8e2205aa](https://user-images.githubusercontent.com/19617248/160961824-3abe3958-c1ee-4088-95df-8b361fb06ebf.gif)


## Implementation description. How did you make this change?
Created a custom check in information card and passed this into the weekly calendar components (made reusable). 
This should support functionality to: 
* View all checkins (as admin)
* Add/remove themselves as a volunteer


## Steps to test
1. Sign in as an admin
2. POST a bunch of checkins so you can populate the page
3. Check you can view everything correctly

## Checklist
